backend/requirements.txt

```diff
@@ -10,6 +10,5 @@ pycodestyle==2.7.0
 pyflakes==2.3.1
 pytz==2021.1
 sqlparse==0.4.1
-toml==0.10.2
```

Comment:
what is this toml for ? I know what it is but where are you planning to use them ?

Reply:
This has been removed

Reply:
Ahh Nice work!

Reply:
nvm I think we will be needing it for autopep8